### PR TITLE
Added FSEP paper

### PR DESCRIPTION
**Pull request by quintessence**
Initial commit for FSEP paper

### CODE REVIEW

File: content/en/docs/papers/fsep/_index.md [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+
+This document proposes a solution to this challenge that will attempt to normalize moderation standards to increase safety and lessen the impact of poor behavior in the federated network. 
+
+
+## Challenge 
+The Fediverse is [defined](https://en.wikipedia.org/wiki/Fediverse) as (a portmanteau of "federation" and "universe"), an ensemble of federated (i.e., interconnected) servers that are used for web publishing (i.e., social networking, microblogging, blogging, or websites) and file hosting, which, while independently hosted, can communicate with each other. 
+
+There have been several protocols that have been used to achieve federation between servers. This document will focus on [Activity Pub](https://www.w3.org/TR/activitypub/#social-web-working-group), an officially W3C-recognized standard based on the Activity Streams 2.0 format.  
+
+The rise of the Activity Pub protocol has spawned several independent projects capable of speaking to each other, i.e., Misskey, Mastodon, Funkwhale, and  PeerTube, to name a few.
```
Comment (ThisIsMissEm) [inline content/en/docs/papers/fsep/_index.md:49]:
```suggestion
The rise of the Activity Pub protocol has spawned several independent projects capable of speaking to each other, i.e., Misskey, Mastodon, Funkwhale, and PeerTube, to name a few. 
```

You have a double space in here

File: content/en/docs/papers/fsep/_index.md [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+To prioritize this [collaboration](https://codeberg.org/oliphant/blocklists) and make it easier to defederate with sites with a  history of poor behavior, The Bad Space was created, a searchable database populated with sites that multiple instances have blocked. While still in alpha, this site provides new opportunities to enhance safety and abuse mitigation and improve onboarding and blocklist maintenance by providing an easy way to keep blocklists up to date from a central location through dynamically created export lists and public API that can be used to search the database programmatically.
+
+These features can be integrated into existing platforms to enhance the overall experience on Activity Pub-enabled applications by automatically limiting the opportunity for bad-faith actors to interact.  
+
+
+## Functional Requirements
+
+### Service Feature Integration 
+
+The first step in this process is to make defederated systems aware of the features provided by services such as The Bad Space, specifically its public search API and dynamically created and updated blocklists.
```
Comment (ThisIsMissEm) [inline content/en/docs/papers/fsep/_index.md:74]: Did you mean "defederate" or perhaps "decentralised" or "federated" here?

File: content/en/docs/papers/fsep/_index.md [hunk 3]
```diff
@@ -0,0 +1,218 @@
...
+- Exposing users to harmful content as means of filtering. 
+- Each problematic site must be manually added 
+- No mechanism for automation, which requires blocklists to be manually  maintained and updated 
+- No tool to validate connection requests at the user level for harmful content
+- Underdeveloped blocklist maintenance options to fine-tune what sites can communicate based on the administrator's discretion.
+
+Integration of The Bad Space’s features into backend functionality will improve  not only the administration experience but the individual users as well by: 
+
+- Allowing blocklists to be automatically imported during onboarding dramatically reduces the opportunity to be exposed to harmful content unnecessarily. 
+- The ability to request an updated blocklist or automate the process to check on its own periodically.
```
Comment (ThisIsMissEm) [inline content/en/docs/papers/fsep/_index.md:94]: Aside: Would be interested in whether these can be achieved using existing APIs or if additional work is needed. 

I'm currently working on an PR to add webhooks to domain blocks in Mastodon, and [have also asked about](https://discord.com/channels/231908446830723072/1061480803344855140/1135549584668635137) (on discord) the ability to learn whether a user account signed in via OAuth 2.0 has moderation abilities on that instance. These may help with your endeavours.

File: content/en/docs/papers/fsep/_index.md [hunk 4]
```diff
@@ -0,0 +1,218 @@
...
+
+### Onboarding
+
+The administration onboarding process can be enhanced by leveraging the additions to the back end detailed in the Service Feature Integration. 
+
+| ID | FEATURE | PRIORITY |
+|:--|:--|:--|
+| 1.1 | At setup time, administrators should be able to upload a deny list or select a default to deny list provided. For MVP, only one deny list provider needs to be supported in this list (The Bad Space). Ultimately, the goal is to allow users to choose from one of several vetted tier 0 lists. | P1 |
+
+| 1.2 | At setup time, administrators should be able to select whether or not they want the default deny list to auto-update. If the admin selects “yes,” the list should attempt to auto-update once every 24 hrs.  | P1 |
```
Comment (ThisIsMissEm) [inline content/en/docs/papers/fsep/_index.md:108]: How does this relate to the PR on Mastodon regarding "configure blocklists at setup time" (not linking directly here, as to not draw attention to this before you're ready, but feel free to ask me for a link on discord)

File: content/en/docs/papers/fsep/_index.md [hunk 5]
```diff
@@ -0,0 +1,218 @@
...
+| 2.4 | Overrides: It should be possible for an admin to add a manual override for a deny list. The manual override should be applied over all deny lists. | P1 |
+| 2.5 | Remove Instance from deny list: It should be possible for an administrator to remove instances where deemed appropriate. These changes only apply to local deny lists and will not affect subscribed deny lists.  | P1 |
+
+### Moderation Data
+
+Moderation capabilities will be expanded to provide more context and information to make informed decisions. 
+
+| ID | FEATURE | PRIORITY |
+|:--|:--|:--|
+| 3.1 | It should be possible for admins to see how many of their users’ followers / following would be impacted by a deny list. For example: *“When you subscribe to [The Bad Space], [Number] of your users will no longer be able to communicate with [number] followers and [number] following.”*| P1 |
```
Comment (ThisIsMissEm) [inline content/en/docs/papers/fsep/_index.md:128]: There's a new API in Mastodon 4.2 that calculates this, though I'm not sure if it's a public-use API or something internal to the admin panel.

File: content/en/docs/papers/fsep/_index.md [hunk 6]
```diff
@@ -0,0 +1,218 @@
...
+
+An additional UI component can be added to this flow to inform the administrator of the features detailed in the Onboarding section.
+
+The first panel of the new component will describe what The Bad Space is, what it does, and how it will be used. It will also contain an element to confirm approval by the administrator to use The Bad Space’s feature set in the platform's experience. If the administrator declines, the flow will continue with the default onboarding process.
+
+![ONBOARDING BLOCKLIST PANEL](Blocklist%20Panel%20One@1x.png)
+
+Upon consenting to the system to use the additional features, a second panel will be revealed to ask if they would like the dynamic blocklist automatically imported from The Bad Space to populate the system's blocklist with the option to check for updates.
```
Comment (ThisIsMissEm) [inline content/en/docs/papers/fsep/_index.md:153]: The word "consenting" here is a bit odd — I know what you mean, but maybe "connecting" or "integrating" might be better?

File: content/en/docs/papers/fsep/_index.md (hunk 1 above)
Comment (are0h) [reply content/en/docs/papers/fsep/_index.md:49]: 👍🏾

File: content/en/docs/papers/fsep/_index.md (hunk 2 above)
Comment (are0h) [reply content/en/docs/papers/fsep/_index.md:74]: Oops, yes. Either works.

File: content/en/docs/papers/fsep/_index.md (hunk 3 above)
Comment (are0h) [reply content/en/docs/papers/fsep/_index.md:94]: Specific implementation is outside the scope of the doc.

File: content/en/docs/papers/fsep/_index.md (hunk 5 above)
Comment (are0h) [reply content/en/docs/papers/fsep/_index.md:128]: Cool.

File: content/en/docs/papers/fsep/_index.md (hunk 6 above)
Comment (are0h) [reply content/en/docs/papers/fsep/_index.md:153]: Hm. The paradigm for the connection between The Bad Space and other platforms is a symbiotic relationship, so the word choice is specific.

I'm going for less techy and more accessible language.

**Pull request merged**